platform/saivpp/vpplib/SwitchStateBaseRoute.cpp

```diff
@@ -66,6 +70,44 @@ void create_route_prefix_entry (
     }
 }
 
+std::string getInterfaceNameFromIP (const std::string &ipAddress) 
```

Comment:
Reinventing the wheel. Please use the implementation here as the basis or modify it for your purpose.
https://github.com/sonic-net/sonic-platform-vpp/blob/e36ae546c3c9861cde3a99be8ce3fda0aa08bb00/platform/saivpp/vpplib/SwitchStateBaseRif.cpp#L87